cmd/native-ika/cli.go

```diff
@@ -53,9 +61,24 @@ func CmdStart() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			c := sui.NewSuiClient(os.Getenv(PeraChain)).(*sui.Client)
+
+			signer, err := native.CreateSigner(os.Getenv(PeraSignerMnemonic))
+			if err != nil {
+				return err
+			}
+
 			logger := log.With().Str("module", "native").Logger()
 			ctx := cmd.Context()
-			idx, err := native.NewIndexer(ctx, b, logger, minimumBlockHeight)
+
+			pc, err := native.NewIkaClient(c, signer,
+				os.Getenv(PeraNativeLcPackage), os.Getenv(PeraNativeLcModule),
+				os.Getenv(PeraNativeLcFunction), os.Getenv(GasAddress), os.Getenv(GasBudget))
```

Comment:
could you check if the environment is properly set based on the file? You can do it by copying `.env.example` to `.env` and run a command - it will print the env variables (see line 42)